lib/v2/penguin-random-house/articles.js

```diff
@@ -0,0 +1,28 @@
+const utils = require('./utils');
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const link = 'https://www.penguinrandomhouse.com/articles';
+    const res = await got(link);
+    const $ = cheerio.load(res.data);
+
+    const itemArray = $('.archive-module-half-container,.archive-module-third-container')
+        .map(function () {
+            return {
+                url: $(this).find('a').attr('href'),
+                title: $(this).find('.archive-module-text').first().text(),
+                pubDate: new Date(),
```

Comment:
Don't add a pubDate if there isn't a date.
https://docs.rsshub.app/joinus/pub-date.html

The post date is available from 
![image](https://user-images.githubusercontent.com/11386903/180604946-8816adb9-8e1f-438c-a4dc-c5ab1b0565e7.png)